### DISCUSSION TITLE
How to view docs for mobx 5?

### DISCUSSION THREAD

**Discussion by rbabayoff** [mobxjs/mobx#2673] [General] [answered]
Hey, not ready to upgrade to mobx6, as I'm sure others are too. Documentation changed significantly in mobx 6. How do I access / or locally produce the documentation for mobx 5? I'm referring to the docs in the [mobx.js.org](https://mobx.js.org/) site.

Thanks!

**Comment by mweststrate** [comment]
Did you try searching existing issues/ questions first?

**Reply by rbabayoff** [reply]
Yes, I did.

**Reply by mweststrate** [reply]
See https://github.com/mobxjs/mobx/issues/2520. It's also linked from https://mobx.js.org/about-this-documentation.html

**Comment by danielkcz** [comment, marked as answer]
I've added a link to old docs to the current docs.
